docs/asset-store/11-04-minio.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Minio
+type: Metrics
+---
+
+As an external, open-source file storage solution, Minio exposes its own metrics. See the [official documentation](https://github.com/minio/minio/tree/master/docs/metrics) for details. The Asset Stores comes with a preconfigured ServiceMonitor CRD that scraps those metrics into Prometheus. Using the metrics, you can create your own Grafana dashboard or reuse the dashboard that is already prepared.
```

Comment:
`preconfigured ServiceMonitor CRD that scraps those metrics into Prometheus`
=
`preconfigured ServiceMonitor CR so that Prometheus knows to scrap those metrics`